src/lsm/lsm_tree.c

```diff
@@ -455,7 +455,7 @@ __lsm_tree_open_check(WT_SESSION_IMPL *session, WT_LSM_TREE *lsm_tree)
 		    "must be at least %" PRIu64 " (%" PRIu64 "MB)",
 		    S2C(session)->cache_size,
 		    S2C(session)->cache_size / WT_MEGABYTE,
-		    required, required / WT_MEGABYTE);
+		    required, (required + (WT_MEGABYTE - 1))/ WT_MEGABYTE);
```

Comment:
math.h offers a ceil() function. Can we do either of the two:
1. include math.h and use ceil() ( *I am not sure if we want to add a header file just to use this* )
2. Write a CEIL(x) macro to round up x to nearest integer >= x

Reply:
To use ceil is making code complex, as it is flagging bad-function-cast error as ceil returns double and we expect unsigned long long.

Reply:
Can we get a macro instead then, maybe something like: #define CEIL(X, Y) ((X) + ((Y) - 1) / (Y))